app/views/layouts/docs.html.erb

```diff
@@ -129,7 +129,7 @@
 
             <div class="welcome__content">
               <div class="center">
-                <h3 class="h3 caps secondary block m0">Welcome to</h3>
```

Comment:
This layout was introduced in https://github.com/hackclub/hcb/pull/9953 but I'm not actually sure it makes use of this code. I chose to update it anyway so both remain consistent.